### PR TITLE
[Frontend_v2] Improvement in UI

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
**Changes Introduced**

- Message display on no hosted challenge modified.

- Participated challenges tab will no longer will have extra headings on the top.

- UI for Analytics tab improved.

GIF of changes:

Frontend_v2_enhancement

### CODE REVIEW

File: frontend_v2/src/app/components/analytics/host-analytics/host-analytics.component.scss [hunk 1]
```diff
@@ -63,4 +63,6 @@ h4.analytics-details-head {
   }
 }
 
-
+.ev-md-container {
+  margin-top: -20px;
```
Comment (deshraj) [inline frontend_v2/src/app/components/analytics/host-analytics/host-analytics.component.scss:67]: Umm, seems like something is off here. We should not be using the `-20px` or any px related css as @Shekharrajak pointed out in the comment.

File: frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html [hunk 2]
```diff
@@ -57,7 +57,7 @@
       <mat-tab label="My Upcoming Challenges ({{ filteredUpcomingChallenges.length }})">
         <div class="challenge-page-title" id = "ongoing-challenges" ><strong class="text-med-black fs-18">My Hosted Challenges</strong></div>
         <div *ngIf="!filteredUpcomingChallenges.length">
-          You haven't hosted any challenge. Please <a [routerLink]="hostTeamsRoutePathCommon" class="highlight-link">click here</a> to host one!
+          You haven't any upcoming hosted challenge. Please <a [routerLink]="hostTeamsRoutePathCommon" class="highlight-link">click here</a> to host one!
```
Comment (deshraj) [inline frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html:60]: Please change the message to `No upcoming hosted challenges found. Please ....`

File: frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html [hunk 3]
```diff
@@ -70,7 +70,7 @@
       <mat-tab label="My Past Challenges ({{ filteredPastChallenges.length }})">
         <div class="challenge-page-title" id = "ongoing-challenges" ><strong class="text-med-black fs-18">My Hosted Challenges</strong></div>
         <div *ngIf="!filteredPastChallenges.length">
-          You haven't hosted any challenge. Please <a [routerLink]="hostTeamsRoutePathCommon" class="highlight-link">click here</a> to host one!
+          You haven't any past hosted challenge.
```
Comment (deshraj) [inline frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html:73]: Please change the message to `No past hosted challenges found. Please .....`

Comment (Kajol-Kumari) [reply frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html:73]: Hey @deshraj I think we shouldn't show the link for hosting a challenge here as hosting a past challenge doesn't make sense.

**Pull request merged**